packages/react-generator/utils/logDiffResult.js

```
@@ -0,0 +1,32 @@
+export const logDiffResult = (result) => {
+  console.log(
```

ESLint

[error] Unexpected console statement.

(no-console)